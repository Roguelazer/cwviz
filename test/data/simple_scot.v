module First (x);
    input x;

    wire z[4:0];
    and2 a00(z[1], z[2], z[3]);
    and2 a01(z[1], z[2], z[4]);
endmodule

module Second;
    wire y[4:0];
    and2 a00(y[1], y[2], y[3]);
    and2 a01(y[1], y[2], y[3]);
endmodule

module Third;
    wire x[4:0];
    buffer b0(x[0], x[1]);
    buffer b1(x[1], x[2]);
    buffer b2(x[2], x[3]);
    buffer b3(x[3], x[4]);
endmodule

module buffer(in, out);
    input in;
    output out;
endmodule

module and2(a, b, c);
    input a;
    input b;
    output c;
endmodule
