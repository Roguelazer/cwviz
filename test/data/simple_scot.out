z_1.Trise   0.0
z_2.Trise   0.5
z_3.Tfall   1.0
z_4.Tfall   5.0e-1

y_1.Trise   0.0
y_2.Trise   0.0
y_3.Tfall   2.0

x_0.Trise   0.0
x_1.Trise   0.5
x_2.Trise   1.0
x_3.Trise   1.5
x_4.Trise   2.0
